src/test/java/cloud/fogbow/ras/core/processors/FulfilledProcessorTest.java

```diff
@@ -1,76 +1,59 @@
 package cloud.fogbow.ras.core.processors;
 
+import java.util.Properties;
```

Comment:
there are exceptions declared in some method that are never thrown (the IDE can spot these mistakes if configured)

Reply:
Unused exceptions declaration has been removed.